gitdm/models.py

```
@@ -40,28 +45,32 @@
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS: list[str] = []  # when creating superuser via createsuperuser
+    REQUIRED_FIELDS: list[str] = []
 
     class Meta:
         verbose_name = "User"
         verbose_name_plural = "Users"
         indexes = [
             models.Index(fields=["email"]),
             models.Index(fields=["date_joined"]),
+            models.Index(fields=["is_doctor", "is_patient"]),
         ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)